Upstream-Status: Pending

automake version 1.11.2 has made use of dir variables more strict.
the use of pkglibdir with DATA var results in automake errors.
  This commits uses pkgdatadir var instead of pkglibdir to avoid 
the strict check errors.

Fixes this error:
service-type-database/Makefile.am:21: `pkglibdir' is not a legitimate directory for `DATA'
autoreconf: automake failed with exit status: 1
ERROR: autoreconf execution failed.

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2011/12/28


Index: avahi-0.6.30/service-type-database/Makefile.am
===================================================================
--- avahi-0.6.30.orig/service-type-database/Makefile.am
+++ avahi-0.6.30/service-type-database/Makefile.am
@@ -18,13 +18,12 @@
 EXTRA_DIST=build-db.in service-types
 
 pkgdata_DATA=service-types
-pkglib_DATA=
 
 if HAVE_PYTHON
 if HAVE_GDBM
 
 noinst_SCRIPTS=build-db
-pkglib_DATA+=service-types.db
+pkgdata_DATA+=service-types.db
 
 build-db: build-db.in
 	$(AM_V_GEN)sed -e 's,@PYTHON\@,$(PYTHON),g' \
@@ -41,7 +40,7 @@ endif
 if HAVE_DBM
 
 noinst_SCRIPTS=build-db
-pkglib_DATA+=service-types.db.pag service-types.db.dir
+pkgdata_DATA+=service-types.db.pag service-types.db.dir
 
 build-db: build-db.in
 	$(AM_V_GEN)sed -e 's,@PYTHON\@,$(PYTHON),g' \
